src/components/challenges/ActiveCard.svelte

```diff
@@ -10,7 +10,7 @@
 
   let challengeDetails = {
     bannerTitle: `Challenge-DF ${curEpoch.id}`,
-    challengeDescription: `Crunch some code and participate in the <strong>weekly</strong> data challenge. </br>Predict the price of Ethereum for a share of <strong>${curEpoch.streams[1].substreams[1].rewards} OCEAN</strong> rewards.`,
+    challengeDescription: `Crunch some code and participate in the active data challenge for <strong>predicting the price of Ethereum</strong>. Data Farming challenges are a substream of Ocean Protocol Active Rewards, running over <strong>periods of 1 week</strong>. You can claim your prize from the <a href="https://df.oceandao.org/rewards">Rewards page</a>.`,
     reward: `${curEpoch.streams[1].substreams[1].rewards} OCEAN in Rewards`,
     buttonText: 'PARTICIPATE',
     deadlineText: 'Enter before ' + getUpcomingFirstWednesdayOfTheMonth().format('DD MMM YYYY'),
```

Comment:
It's not before this day. It's by EoD. Can we adjust this?

Reply:
good point, is there a reason why we are calculating the first Wednesday of the month? I don't think I implemented that one, and this challenge should be weekly.